### ISSUE TITLE
If statements should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#508]
If statements method should be surrounded by a blank line to ease reading (spotting if statements with ease).

Following should report a violation:
```c#
var condition = false;
if (condition)
{
    ...
}
```

While following should NOT report a violation:
```c#
var condition = false;

if (condition)
{
    ...
}
```
